### ISSUE TITLE
[Screen Reader - Sparkline]: NVDA is not giving information that the cell containes Sparklines.

### ISSUE THREAD

**Issue by v-raipra** [nvaccess/nvda#7964]
User Experience:
User with visual impairments will not get the proper information if the NVDA does not speak the proper information.
 
Test Environment:
Build: 17.7806.1000
OS: Win 10 RS PRERELEASE 15008.1000
Tool used: NVDA

Repro Steps: 
Open Excel.
Create a blank workbook.
Create a tabular data in the grid
Get the Focus on 'Excel' Ribbon.
Navigate to 'Insert' Tab and activate it using Enter key
Navigate to "Column Sparkline" button and activate it .
Navigate to the cell that contains "Column Sparkline".
Observe the issue.
Actual:
NVDA is not speaking that the cell contains "Column Sparkline" it only narrate the cell value.
 
Expected:
Narrator should speak that "Contains Column Sparkline" with the address and the value of the cell.
 
Guidelines:
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7Bd8e9eb97-184e-4731-af9a-e74ca265c896%7D&action=view&wdparaid=7509BFA3
 
[New Compressed (zipped) Folder.zip](https://github.com/nvaccess/nvda/files/1702861/New.Compressed.zipped.Folder.zip)

**Comment by seanbudd** [comment]
is this still an issue?

**Comment by Adriani90** [comment]
yes this is still reproducible. Actually the sparkline of the cell can be reported only via object navigation, it seems to become a child object of the cell but it is not reported when arrowing through the map. Tested both with and without UIA.
This should be optionally reported when arrowing (e.g. a combo box in document formatting settings called in-cell diagrams" which contains speech, braille, both or off)
